### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/linkerd2/commit/0d56327e6ffb14d1e389a9b33bd95718988d03cd#commitcomment-51603624

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/linkerd2/commit/03a9c310aa5612f2356d04b61a1c7e226a48d096

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -674,7 +674,7 @@ the robustness of the opaque transport.
 * Changed opaque-port transport to be advertised via ALPN so that new proxies
   will not initiate opaque-transport connections to proxies from prior edge
   releases
-* Added inbound proxy transport metrics with `tls="passhtru"` when forwarding
+* Added inbound proxy transport metrics with `tls="passthru"` when forwarding
```
Comment (jsoref) [inline CHANGES.md:677]: ?

File: CHANGES.md [hunk 2]
```diff
@@ -823,7 +823,7 @@ async runtime.
   `linkerd-jaeger` extension is working correctly
 * Added new `linkerd jaeger uninstall` CLI command to print the `linkerd-jaeger`
   extension's resources so that they can be piped into `kubectl delete`
-* Fixed an issue where the `linkerd-cni` daemgitonset may not be installed on all
+* Fixed an issue where the `linkerd-cni` daemonset may not be installed on all
```
Comment (jsoref) [inline CHANGES.md:826]: ?

File: jaeger/cmd/install_test.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ func TestRender(t *testing.T) {
 	// pin values that are changed by render functions on each test run
 	defaultValues := map[string]interface{}{
 		"webhook": map[string]interface{}{
-			"keyPEM":   "test-webhhook-key-pem",
+			"keyPEM":   "test-webhook-key-pem",
```
Comment (jsoref) [inline jaeger/cmd/install_test.go:16]: ?

File: jaeger/injector/mutator/webhook.go [hunk 4]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/linkerd/linkerd2/controller/k8s"
 	"github.com/linkerd/linkerd2/controller/webhook"
 	"github.com/linkerd/linkerd2/jaeger/pkg/labels"
-	l5dLables "github.com/linkerd/linkerd2/pkg/k8s"
+	l5dLabels "github.com/linkerd/linkerd2/pkg/k8s"
```
Comment (jsoref) [inline jaeger/injector/mutator/webhook.go:13]: probably harmless

File: pkg/tree/tree_test.go [hunk 5]
```diff
@@ -34,7 +34,7 @@ func TestTreeGetString(t *testing.T) {
 		},
 		{
 			values,
-			[]string{"global", "namespac"},
+			[]string{"global", "namespace_"},
```
Comment (jsoref) [inline pkg/tree/tree_test.go:37]: ??

File: test/integration/opaqueports/opaque_ports_test.go [hunk 6]
```diff
@@ -94,7 +94,7 @@ func TestOpaquePorts(t *testing.T) {
 	// Wait for slow-cookers to start sending requests
 	time.Sleep(20 * time.Second)
 
-	t.Run("expect absent HTTP outbound requests for opaque-pod slow clooker", func(t *testing.T) {
+	t.Run("expect absent HTTP outbound requests for opaque-pod slow cooker", func(t *testing.T) {
```
Comment (jsoref) [inline test/integration/opaqueports/opaque_ports_test.go:97]: ?

File: CHANGES.md (hunk 1 above)
Comment (alpeb) [reply CHANGES.md:677]: This seems fine

File: CHANGES.md (hunk 2 above)
Comment (alpeb) [reply CHANGES.md:826]: `daemonset` is correct :+1:

File: jaeger/cmd/install_test.go [hunk 7]
```diff
@@ -13,7 +13,7 @@ func TestRender(t *testing.T) {
 	// pin values that are changed by render functions on each test run
 	defaultValues := map[string]interface{}{
 		"webhook": map[string]interface{}{
-			"keyPEM":   "test-webhhook-key-pem",
+			"keyPEM":   "test-webhook-key-pem-ignored",
```
Comment (alpeb) [inline jaeger/cmd/install_test.go:16]: Something odd happened here

File: pkg/tree/tree_test.go (hunk 5 above)
Comment (alpeb) [reply pkg/tree/tree_test.go:37]: Yep, this was weird

**Review by alpeb — COMMENTED** [review]
Thanks @jsoref, I've added a couple of comments below

File: jaeger/cmd/install_test.go (hunk 3 above)
Comment (alpeb) [reply jaeger/cmd/install_test.go]: this is fine :+1:

File: test/integration/opaqueports/opaque_ports_test.go (hunk 6 above)
Comment (alpeb) [reply test/integration/opaqueports/opaque_ports_test.go:97]: this is fine :+1:

File: jaeger/cmd/install_test.go (hunk 7 above)
Comment (jsoref) [reply jaeger/cmd/install_test.go:16]: Oh, I meant to flag this one. The naive fix causes the tests to fail.

Comment (jsoref) [reply jaeger/cmd/install_test.go]: *nearly* the same value.

**Review by alpeb — APPROVED** [review]
Thanks @jsoref , changes look good to me :+1:

**Review by kleimkuhler — APPROVED** [review]
Thanks @jsoref!

**Pull request merged**